### ISSUE TITLE
Error message. Opening and saving. No preview of image.

### ISSUE THREAD

**Issue by zb13y** [mypaint/mypaint#374]
```python
Mypaint version: 1.2.0-beta.0+gitexport.adc9031
System information: Windows-7-6.1.7601-SP1
Traceback (most recent call last):
  File "C:\Program Files\MyPaint\share\mypaint/gui/filehandling.py", line 492, update_preview_cb(self=<gui.filehandling.FileHandler object>, file_chooser=<FileChooserDialog object at 0xf970730 (GtkFileChooserDialog at 0xabc08d0)>, preview=<Image object at 0xf9775a0 (GtkImage at 0xf761660)>)
                filename = filename.decode('utf-8')
                pixbuf = helpers.freedesktop_thumbnail(filename)
                if pixbuf:
  variables: {'helpers.freedesktop_thumbnail': ('global', <function freedesktop_thumbnail at 0x0000000009fa5488>), 'pixbuf': (None, []), 'filename': ('local', u'C:\\Users\\\u0410\u0437\u0430\u043c\u0430\u0442\\Documents\\MyPaint\\scrap001_a.ora')}
  File "C:\Program Files\MyPaint\share\mypaint/lib/helpers.py", line 167, freedesktop_thumbnail(filename=u'C:\\Users\\\u0410\u0437\u0430\u043c\u0430\u0442\\Documents\\MyPaint\\scrap001_a.ora', pixbuf=None)
        uri = _filename2uri_freedesktop_canon(os.path.abspath(filename))
        logger.debug("thumb: uri=%r", uri)
  variables: {'os.path.abspath': ('global', <function abspath at 0x00000000024c29b0>), '_filename2uri_freedesktop_canon': ('global', <function _filename2uri_freedesktop_canon at 0x0000000009fa5668>), 'uri': (None, []), 'filename': ('local', u'C:\\Users\\\u0410\u0437\u0430\u043c\u0430\u0442\\Documents\\MyPaint\\scrap001_a.ora')}
  File "C:\Program Files\MyPaint\share\mypaint/lib/helpers.py", line 308, _filename2uri_freedesktop_canon(path=u'C:\\Users\\\u0410\u0437\u0430\u043c\u0430\u0442\\Documents\\MyPaint\\scrap001_a.ora', encoding='mbcs')
        path_bytes = path.encode(encoding)
        return GLib.filename_to_uri(path_bytes, None)
  variables: {'None': ('builtin', None), 'path_bytes': ('local', 'C:\\Users\\\xc0\xe7\xe0\xec\xe0\xf2\\Documents\\MyPaint\\scrap001_a.ora'), 'GLib.filename_to_uri': ('global', gi.FunctionInfo(filename_to_uri))}
Error: g-invoke-error-quark: Could not locate g_filename_to_uri: 'g_filename_to_uri': Не найдена указанная процедура. (1)
```

Always the same error message. Even if i only go to menu 'file-open' and i'll tick the file of img without opening and then go to another programm, for example to my browser, there will be these error message. I cant see preview of images, like in 1.0. There are only name of the file, without preview.

**Comment by achadwick** [comment]
Reopening this: I'm finding that it's showing up in the "mypaint-w64" build (and on the MSYS2 command line in "testing mode"), but not in the "mypaint-w32" build.

**Comment by achadwick** [comment]
This should be fixed with 193264857ced956ebb591e3451b428b707414cf0, released as v1.2.0-beta.0.73 in <https://github.com/mypaint/mypaint/releases/tag/v1.2.0-beta.0>. @zb13y: please can you download and test? Thanks!
